tools/vpn/vpn_softetherbridge.md

```diff
@@ -99,33 +106,35 @@ VPN Server/BRIDGE>CascadeOnline mybridge   <---
 
 If you need to connect the machines in VPN from the machine where `vpnbridge` is running,
 
-create tap interface on step 6 in above,
+Create tap interafce, so it can communicate with local network,
 
 ```
 VPN Server>BridgeCreate BRIDGE /DEVICE:svpn /TAP:yes      <----
 ```
 
-Install net-tools and bridged-utils to create a bridged interface.
+Install `bridged-utils` to create a bridged interface,
```

Comment:
Each step should be a sentence, ending with a period (a/k/a full-stop) `.`. They should not end with commas, as this PR is doing.

Reply:
Thank you for the sugesstion!